### PR TITLE
Feature/movement

### PR DESCRIPTION
**Pull request by alarson892**
Basic movement functionality

### CODE REVIEW

File: summer-23-unity/Assets/Scripts/PlayerMovement.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+        if (move != Vector3.zero)
+        {
+            gameObject.transform.forward = move;
+        }
+
+        // Changes the height position of the player..
+        if (Input.GetButtonDown("Jump") && groundedPlayer)
+        {
+            playerVelocity.y += Mathf.Sqrt(jumpHeight * -3.0f * gravityValue);
```
Comment (clarson99) [inline summer-23-unity/Assets/Scripts/PlayerMovement.cs]: what is '-3.0f'?  Is that a magic constant?  It probably belongs up above with the other constants so i has a descriptive name.

**Review by clarson99 — CHANGES_REQUESTED** [review]
Can you fix the issue with the constant and add some docs about how you are doing movement?  Something like ./docs/movement.md would be a good start.

**Comment by alarson892** [comment]
Requested changes made

**Review by clarson99 — APPROVED** [review]
lgtm

**Pull request merged**